src/mokka/src/main/resources/templates/index.html

```diff
@@ -20,27 +21,19 @@
 		<md-toolbar>
 	      <div class="md-toolbar-tools">
 	        <h2>
-	          <span>Mokka</span>
-	          <span th:if="${path == 'login'}">&nbsp;>&nbsp;Login</span>
-	          <span th:if="${path == 'log'}">&nbsp;>&nbsp;Logs</span>
-	          <span th:if="${path == 'configuration'}">&nbsp;>&nbsp;Configuration</span>
+	          <div id="logoMokka"></div>
 	        </h2>
 
 			<md-menu sec:authorize="hasRole('ROLE_USER')">
-			  <md-button aria-label="Navigation" class="md-icon-button" ng-click="$mdOpenMenu($event)">
-			    <ng-md-icon icon="menu" style="fill: #fff"></ng-md-icon>
-			  </md-button>
-			  <md-menu-content width="4">
-			    <md-menu-item th:if="${path != 'log'}">
-			      <md-button th:href="@{/logs}" target="_self">Logs</md-button>
-			    </md-menu-item>
-			    <md-menu-item th:if="${path != 'configuration'}">
-			      <md-button th:href="@{/configurations}" target="_self">Configuration</md-button>
-			    </md-menu-item>
-			    <md-menu-item th:if="${path != 'user'}" sec:authorize="hasAnyRole('ROLE_ADMIN', 'ROLE_USER_ADMIN')">
-			      <md-button th:href="@{/users}" target="_self">Users</md-button>
-			    </md-menu-item>
-			  </md-menu-content>
+			  <md-content width="4">
+			    <md-nav-bar>
+			      <md-button th:href="@{/logs}" target="_self" th:classappend="${#httpServletRequest.requestURI.contains('logs')} ? 'active' : 'white-text'">Logs</md-button>
```

Comment:
could we please move `httpServletRequest` related part as one common function `isActive`? You could also consider to use `https://code.angularjs.org/1.5.3/docs/api/ng/service/$location` then.